Getting-Started.md

```diff
@@ -225,4 +242,36 @@ In this example, the first transaction will be `approve` transaction that is goi
 
 The second transaction will call the service contract and execute the `buySomeService` function.
 
+___
+
+
+### 10. Invoice/Receipt configuration
+
+This section describes how one can activate and configure invoice/receipt feature so his customers can receive emails.
+
+* ##### Activation steps 
+
+1. You have to provide onboarding team a valid email [(receiptEmail)](https://github.com/LimePay/docs/blob/master/API-Documentation.md#vendor) from which LimePay will send invoices/receipts to your shoppers.   
+For automatically send invoice/receipt on successful payment you have two options:   
+
+	1) Tell onboarding team to activate automatic invoice feature
+	2) Tell onboarding team to activate automatic receipt feature
+  
+**Important!**  You can manually send an invoice via [rest API call](#TODOLINK), but a receipt can be send only if the **automatic receipt** feature is enabled.
+
+
+* ##### Configuration steps
+
+**Important** If you do not provide onboard team some configuration the default one will be used.
```

Comment:
If you do not provide the on-boarding team with the configuration that you want, the default one will be used.